host-services/server.go

```diff
@@ -81,17 +87,39 @@ func (h *HostServicesServer) handleRPC(msg *nats.Msg) {
 		metadata[k] = v[0]
 	}
 
+	ctx := otel.GetTextMapPropagator().Extract(context.Background(), propagation.HeaderCarrier(msg.Header))
+
+	_, span := h.tracer.Start(ctx, "host services call",
+		trace.WithSpanKind(trace.SpanKindClient),
+		trace.WithAttributes(
+			attribute.String("workload_id", vmID),
+			attribute.String("service", serviceName),
+			attribute.String("method", method),
+		))
+	defer span.End()
+
+	span.AddEvent("RPC request")
```

Comment:
typically when we add events we use a past tense form. This way I can tell the difference between whether this event marks the beginning or the end of an RPC request.

Reply:
If we have an "RPC Request Began" and "RPC Request Completed" event, and then we flag the span as error for a fail, it becomes a pinch easier to find what we're looking for in a grafana search